### DISCUSSION TITLE
Registro elettronico giua@school

### DISCUSSION THREAD

**Discussion by buttifranco** [iisgiua/giuaschool#560] [Richieste di aiuto] [answered]
Qualcuno può aiutarmi a capire questo registro elettronico?

**Comment by trinko** [comment]
Anzi tutto leggi la [documentazione](https://iisgiua.github.io/giuaschool-docs/)
Poi fai domande specifiche su ciò che vuoi sapere e cercherò di rispondere.

**Comment by buttifranco** [comment]
Buongiorno, mi hanno installato il RE Giua, con l’incarico di caricare i dati. Sinceramente non sono molto preparato per ciò, mi può dare una mano? Il mio numero è XXXX. Grazie

 

Da: Antonello Dessì ***@***.***> 
Inviato: sabato 5 agosto 2023 13:07
A: iisgiua/giuaschool ***@***.***>
Cc: buttifranco ***@***.***>; Author ***@***.***>
Oggetto: Re: [iisgiua/giuaschool] Registro elettronico ***@***.*** (Discussion #560)

 

Anzi tutto leggi la documentazione <https://iisgiua.github.io/giuaschool-docs/> 
Poi fai domande specifiche su ciò che vuoi sapere e cercherò di rispondere.

**Comment by trinko** [comment, marked as answer]
Anzi tutto evita di scrivere dati personali in un'area pubblica.
Comunque, questo non è un servizio di assistenza: ti posso spiegare come fare, poi se hai problemi spieghi bene in dettaglio  il problema e vediamo se si riesce a risolvere.

Come ti dicevo, devi partire dalla documentazione dove trovi il capitolo sull'importazione dei dati. Inizia ad usare i file di prova che puoi scaricare dalla pagina: [importazione degli utenti](https://iisgiua.github.io/giuaschool-docs/configure.html#importazione-degli-utenti)

Fai qualche prova con quei dati poi prova a modificarli creando i file di importazione reali.

I file di importazione sono in formato CSV, che puoi creare e modificare con Excel o OpenOffice/LibreOffice. Non usare lettere accentate, creano solo problemi: usa la vocale seguita dall'apostrofo (**Nicolo'** e non **Nicolò**).

**Comment by ghost** [comment]
@buttifranco Buongiorno! Avrei bisogno di parlarti, in merito a questa richiesta. Posso parlarti in privato? Mi scrivi alla mia mail: padovesi.roberto@gmail.com? Grazie!
